monkey/common/agent_configuration/default_agent_configuration.py

```diff
@@ -27,7 +27,7 @@
     "other_behaviors": {"readme": True},
 }
 
-PAYLOAD_CONFIGURATION = tuple([PluginConfiguration(name="ransomware", options=RANSOMWARE_OPTIONS)])
+PAYLOAD_CONFIGURATION = {"ransomware": RANSOMWARE_OPTIONS}
```

Comment:
I expect that the agent will need to be updated to cope with this change. Have you tried running the ransomware module?